grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandlerBuilder.java

```diff
@@ -60,7 +60,7 @@ public final class UnframedGrpcErrorHandlerBuilder {
      * for encoding the error responses. If messages or message types have
      * already been registered, calling this method will result in an
      * {@link IllegalStateException}. If nothing is specified,
-     * {@link UnframedGrpcErrorHandlers#ERROR_DETAILS_MARSHALLER} is used as
+     * {@link JsonUnframedGrpcErrorHandler#ERROR_DETAILS_MARSHALLER} is used as
      * default json marshaller.
```

Comment:
We can't link non-public API to Javadoc of public API. Should we simply say?
```suggestion
     * the default JSON marshaller is used.
```